Lib/dataclasses.py

```diff
@@ -198,10 +198,13 @@ def __repr__(self):
 
 class _InitVarMeta(type):
     def __getitem__(self, params):
-        return self
+        return InitVar(params)
 
 class InitVar(metaclass=_InitVarMeta):
-    pass
+    __slots__ = ('type', )
+
+    def __init__(self, type):
+         self.type = type
```

Comment:
Found it: this line has 5 space indentation.

Reply:
ohhh, little bastard. thanks, should be fixed now :)